packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -200,14 +207,25 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
     }
 
     private renderQueryList = (listProps: QueryListRendererProps<T>) => {
-        const { disabled, popoverContentProps = {}, popoverProps = {} } = this.props;
+        const { disabled, placeholder, popoverContentProps = {}, popoverProps = {} } = this.props;
         const { handleKeyDown, handleKeyUp } = listProps;
 
         const popoverRef =
             this.props.popoverRef === undefined
                 ? this.refHandlers.popover
                 : mergeRefs(this.refHandlers.popover, this.props.popoverRef);
 
+        const input = (
```

Comment:
there's already an `input` ref, could we call this something more specific like `popoverInputForCustomTarget`, and also assign it a unique ref so we consider these cases separately?